modules/persistent-storage-csi-efs-cross-account.adoc

```diff
@@ -0,0 +1,304 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-efs-cross-account_{context}"]
+= AWS EFS CSI cross account support
+
+Cross account support allows you to have an {product-title} cluster in one AWS account and mount your file system in another AWS account using the AWS Elastic File System (EFS) Container Storage Interface (CSI) driver.
+
+[NOTE]
+====
+Both the {product-title} cluster and EFS file system should be in the same region.
```

Comment:
must be in the same region?

Reply:
@gcharot According QE's test procedure: https://docs.google.com/document/d/1-c87alz1p4Q6xUxIEx-aogkocZpaXeLXbN3HVrxLvPo/edit#

Reply:
yes i think this is just an abuse of the word "should". We don't support multi region so I assume "must" is better suited?

Reply:
Correct. Done.